ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRowTest.java

```diff
@@ -149,6 +154,17 @@ public void testUnmap() throws Exception {
     assertEquals(expected, Json.asString(parent));
   }
 
+  @Test
+  public void testUnmapWithoutValue() throws Exception {
+    Map<String, Object> parent = new HashMap<>();
+    Map<String, Object> additionalProperties = new HashMap<>();
+    parent.put("mapField", new HashMap<>(ImmutableMap.of("foo", 3, "bar", 4)));
+    List<Field> bqFields = ImmutableList.of(MAP_FIELD_WITHOUT_VALUE);
+    String expected = "{\"map_field\":[{\"key\":\"bar\"},{\"key\":\"foo\"}]}";
+    TRANSFORM.transformForBqSchema(parent, bqFields, additionalProperties);
+    assertEquals(expected, Json.asString(parent));
```

Comment:
Could you add a test case for the expected structure of the `additional_properties` field when the schema specifies a valueless map, please? For the concrete example of XPI installs, I'm wondering how the value for a specific key can be accessed using `JSON_EXTRACT` udf.